File: - ToCode/Android/Apps/CannonGame/MainActivity.java
// MainActivity displays the MainActivityFragment
package com.deitel.cannongame;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {
   // called when the app first launches
   @Override
   protected void onCreate(Bundle savedInstanceState) {
      super.onCreate(savedInstanceState);
      setContentView(R.layout.activity_main);
   }
}

File: @legacy@/crossPlatform/apps/CannonGame/CannonGame.App/MainActivity.cs
using Android.App;
using Android.Content.PM;
using Android.OS;
using Android.Support.V7.App;
using Android.Views;
using ResLayout = AppDevUnited.CannonGame.App.Resource.Layout;

namespace AppDevUnited.CannonGame.App
{
   [Activity(
      Label = "@string/app_name",
      Theme = "@style/AppTheme.NoActionBar",
      MainLauncher = true,
      ScreenOrientation = ScreenOrientation.Landscape,
      WindowSoftInputMode = SoftInput.StateAlwaysVisible)]
   public class MainActivity : AppCompatActivity
   {
      protected override void OnCreate(Bundle savedInstanceState)
      {
         base.OnCreate(savedInstanceState);
         SetContentView(ResLayout.ActivityMain);
      }
   }
}